metrics_sdk/lib/opentelemetry/sdk/metrics/export/periodic_metric_reader.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        # PeriodicMetricReader provides a minimal example implementation.
+        class PeriodicMetricReader < MetricReader
+          def initialize(interval_millis: ENV.fetch('OTEL_METRIC_EXPORT_INTERVAL', 60),
+                         timeout_millis: ENV.fetch('OTEL_METRIC_EXPORT_TIMEOUT', 30),
+                         exporter: nil)
+            super()
+
+            @interval_millis = interval_millis
```

Comment:
What does `_millis` mean? I thought "milliseconds" but after experimenting with this, it seems to be seconds, not milliseconds.

Reply:
Thanks, the name is indeed confusing. It should be seconds. I have updated the name to export_interval and export_timeout.

Reply:
Awesome, thanks! :)